osu.Game.Rulesets.Osu/Objects/Drawables/DrawableRepeatPoint.cs

```diff
@@ -27,16 +27,14 @@ public DrawableRepeatPoint(RepeatPoint repeatPoint, DrawableSlider drawableSlide
             AutoSizeAxes = Axes.Both;
             Blending = BlendingMode.Additive;
             Origin = Anchor.Centre;
-            Scale = new Vector2(0.5f);
-
             Children = new Drawable[]
             {
                 new SpriteIcon
                 {
                     Icon = FontAwesome.fa_eercast,
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
-                    Size = new Vector2(32),
+                    Size = new Vector2(32 * repeatPoint.Scale),

```

Comment:
Why not set this scale as the DrawableRepeatPoint's scale, rather than just for the icon?